helm/chart-operator/templates/priorityclass.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if .Values.e2e }}
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
```

Comment:
Even though it's e2e-only resource, I would like to put labels as we prefer. https://github.com/giantswarm/fmt/blob/master/kubernetes/annotations_and_labels.md#common-labels

Reply:
Good point. Done.